server/backend/db_backend.go

```diff
@@ -25,11 +26,13 @@ import (
 var wei = big.NewInt(1000000000000000000)
 
 type MongoBackend struct {
-	host         string
-	mongo        *mgo.Database
-	mongoSession *mgo.Session
-	goClient     *goclient.Client
-	Lgr          *zap.Logger
+	host            string
+	mongo           *mgo.Database
+	mongoSession    *mgo.Session
+	goClient        *goclient.Client
+	Lgr             *zap.Logger
+	mutex           sync.RWMutex
```

Comment:
A name like `databaseVersionMu` would be more idiomatic and clear about what it applies to.